webui/package-lock.json

```diff
@@ -14,14 +14,15 @@
         "bootstrap": "^4.6.0",
```

Comment:
I think these got in due to the npm update, should be removed?

Reply:
Build is failing without it 🤷